app/pyproject.toml

```diff
@@ -35,6 +35,9 @@ Faker = "^14.2.0"
 factory-boy = "^3.2.1"
 bandit = "^1.7.4"
 
+[tool.poetry.group.dev.dependencies]
```

Comment:
this is the new way of doing dev dependencies. i didn't bother migrating everything over in this PR since we have a separate ticket for that #37